Intersect.Client/Entities/Player.cs

```diff
@@ -1403,6 +1403,20 @@ private void SetTargetBox(Entity en)
             TargetBox?.Show();
         }
 
+        private void AutoTurnToTarget(Entity en)
+        {
+            if (!Globals.Database.AutoTurnToTarget || !Options.Instance.PlayerOpts.EnableAutoTurnToTarget || IsMoving ||
+                Dir == MoveDir)
+            {
+                return;
+            }
+
+            MoveDir = -1;
+            byte directionToTarget = DirectionToTarget(en);
+            Dir = directionToTarget;
+            PacketSender.SendDirection(directionToTarget);
```

Comment:
Just do
```csharp
Dir = DirectionToTarget(en);
PacketSender.SendDirection(Dir);
```

Reply:
btw, ended up requiring it as a variable in the last commit